src/components/shared/Entity/hooks/useEntityWorkflowHydrated.ts

```diff
@@ -12,13 +13,20 @@ import { useEndpointConfig_hydrated } from 'stores/EndpointConfig/hooks';
 //  every use case so some things might slip through - but it is way better
 //  than not checking any of this stuff.
 function useEntityWorkflowHydrated() {
+    const workflow = useEntityWorkflow();
+
     const bindingsHydrated = useBinding_hydrated();
     const detailsHydrated = useDetailsFormStore((state) => state.hydrated);
     const endpointHydrated = useEndpointConfig_hydrated();
 
+    // The page-based derivation create workflow does not involve any stores
+    // with hydrated state.
     return useMemo(
-        () => bindingsHydrated && detailsHydrated && endpointHydrated,
-        [bindingsHydrated, detailsHydrated, endpointHydrated]
+        () =>
+            workflow === 'collection_create'
+                ? true
+                : bindingsHydrated && detailsHydrated && endpointHydrated,
```

Comment:
Nit pick ahead.... can we make this a single if statement still?

`workflow === 'collection_create' || (...)`